src/ext/java/org/opentripplanner/ext/parkAndRideApi/ParkAndRideResource.java

```diff
@@ -37,7 +38,10 @@ public ParkAndRideResource(
     @Deprecated @PathParam("ignoreRouterId") String ignoreRouterId
   ) {
     this.vehicleParkingService = serverContext.graph().getVehicleParkingService();
-    this.graphFinder = new DirectGraphFinder(serverContext.graph());
+
+    // TODO OTP2 - Why are we using the DirectGraphFinder here, not just
+    //           - serverContext.graphFinder(). This needs at least a comment!
+    this.graphFinder = new DirectGraphFinder(serverContext.transitService()::queryStopSpatialIndex);
```

Comment:
This is historical behaviour. We could just use the StopModel directly, if it has a method to query stops within a radius.

Reply:
Ok, I will leave it for now, but add it to the TODOs list.

Reply:
I updated the todo.